xtas/webserver/__main__.py

```diff
@@ -38,6 +38,6 @@
     print("Serving tasks:")
     pprint(list(taskq.tasks.keys()))
 http_server = HTTPServer(WSGIContainer(app))
-http_server.bind(args['--port'], address=args['--host'])
```

Comment:
--host doesn't select a host to listen on (that can only be the host that the server is running on), it selects an interface on that host (i.e. which network adapter). Not sure whether to change the command line argument, the documentation, or nothing at all.